index.js

```diff
@@ -0,0 +1,23 @@
+const elem = document.getElementById('menuMobile');
+const item = document.querySelectorAll('.list-menu-item');
+const logo = document.getElementById('logo');
+const menubars = document.getElementById('menuBars');
```

Comment:
### Optional

I'll mark this as a suggestion rather than a requirement since the last code reviewer missed it so you can implement it if you wish!

You have two references to `menuBars` in your js file in lines 4 and 18. Please be sure to reuse your variables to avoid code repetition 🤓